deploy/docs/V1_Migration_doc.md

```diff
@@ -0,0 +1,176 @@
+## Helm Users
+### Changes
+
+- Falco installation disabled by Default. If you want to enable Falco, modify the `enabled` flag for Falco in `values.yaml` as shown below: 
+```yaml
+falco:
+  ## Set the enabled flag to false to disable falco.
+  enabled: true
+  ```
+
+- Bumped the helm Falco chart version to v `1.1.6` which included a fix to disable the bitcoin/crypto miner rule by default
+
+- Changes in Configuration Parameters
+	- The `values.yaml` file has had several configs moved and renamed to improve usability. Namely, we introduced a new fluentd section into which we moved all of the Fluentd specific configs, while configs for our dependency charts (`prometheus-operator`, `fluent-bit`, `metrics-server`, `falco`) have not changed.
+
+| Old Config 	| New Config 	|
+|:----------------------------------------:	|:-----------------------------------------------------:	|
+| sumologic.eventCollectionEnabled 	| fluentd.events.enabled 	|
+| sumologic.events.sourceCategory 	| fluentd.events.sourceCategory 	|
+| sumologic.logFormat 	| fluentd.logs.output.logFormat 	|
+| sumologic.flushInterval 	| fluentd.buffer.flushInterval 	|
+| sumologic.numThreads 	| fluentd.buffer.numThreads 	|
+| sumologic.chunkLimitSize 	| fluentd.buffer.chunkLimitSize 	|
+| sumologic.queueChunkLimitSize 	| fluentd.buffer.queueChunkLimitSize 	|
+| sumologic.totalLimitSize 	| fluentd.buffer.totalLimitSize 	|
+| sumologic.sourceName 	| fluentd.logs.containers.sourceName 	|
+| sumologic.sourceCategory 	| fluentd.logs.containers.sourceCategory 	|
+| sumologic.sourceCategoryPrefix 	| fluentd.logs.containers.sourceCategoryPrefix 	|
+| sumologic.sourceCategoryReplaceDash 	| fluentd.logs.containers.sourceCategoryReplaceDash 	|
+| sumologic.addTimestamp 	| fluentd.logs.output.addTimestamp 	|
+| sumologic.timestampKey 	| fluentd.logs.output.timestampKey 	|
+| sumologic.verifySsl 	| fluentd.verifySsl 	|
+| sumologic.excludeContainerRegex 	| fluentd.logs.containers.excludeContainerRegex 	|
+| sumologic.excludeHostRegex 	| fluentd.logs.containers.excludeHostRegex 	|
+| sumologic.excludeNamespaceRegex 	| fluentd.logs.containers.excludeNamespaceRegex 	|
+| sumologic.excludePodRegex 	| fluentd.logs.containers.excludePodRegex 	|
+| sumologic.fluentdLogLevel 	| fluentd.logLevel 	|
+| sumologic.watchResourceEventsOverrides 	| fluentd.events.watchResourceEventsOverrides 	|
+| sumologic.fluentd.buffer 	| fluentd.buffer.type 	|
+| sumologic.fluentd.autoscaling.* 	| fluentd.logs.autoscaling.* , fluentd.metrics.autoscaling.* 	|
+| sumologic.k8sMetadataFilter.watch 	| fluentd.logs.containers.k8sMetadataFilter.watch 	|
+| sumologic.k8sMetadataFilter.verifySsl 	| fluentd.logs.containers.k8sMetadataFilter.verifySsl 	|
+| sumologic.k8sMetadataFilter.cacheSize 	| fluentd.metadata.cacheSize 	|
+| sumologic.k8sMetadataFilter.cacheTtl 	| fluentd.metadata.cacheTtl 	|
+| sumologic.k8sMetadataFilter.cacheRefresh 	| fluentd.metadata.cacheRefresh 	|
+| deployment.* 	| fluentd.statefulset.* 	|
```

Comment:
this config has been split to `fluentd.logs.statefulset.* , fluentd.metrics.statefulset.* `